dist/pathfora.js

```diff
@@ -2555,7 +2557,7 @@
 
             // CASE: Content recommendation elements
             case 'data-pfrecommend':
-              if (context.pathfora.acctid === '') {
+              if (pathfora.acctid === '') {
```

Comment:
probably should also test if pathfora.acctid is undefined here as well. if for some reason pathfora is not an object pathfora.acctid will be undefined it wont throw an error. i think. 

granted, edge case since we are invoking the function but /shrug